app/graphicsview.cpp

```diff
@@ -258,22 +258,22 @@ void GraphicsView::dropEvent(QDropEvent *event)
     if (mimeData->hasUrls()) {
         const QList<QUrl> &urls = mimeData->urls();
         if (urls.isEmpty()) {
-            showText(tr("File url list is empty"));
+            showText(tr("The file URI list is empty"));
         } else {
             showFileFromUrl(urls.first(), true);
         }
     } else if (mimeData->hasImage()) {
         QImage img = qvariant_cast<QImage>(mimeData->imageData());
         QPixmap pixmap = QPixmap::fromImage(img);
         if (pixmap.isNull()) {
-            showText(tr("Image data is invalid"));
+            showText(tr("Invalid image data"));
         } else {
             showImage(pixmap);
         }
     } else if (mimeData->hasText()) {
         showText(mimeData->text());
     } else {
-        showText(tr("Not supported mimedata: %1").arg(mimeData->formats().first()));
+        showText(tr("Unsupported media type: %1").arg(mimeData->formats().first()));

```

Comment:
https://en.wikipedia.org/wiki/Media_type or the data within it?

Reply:
It means the MIME-type is not supported.

Reply:
OK, then this is good. MIME-type changed name to "media type" some time ago.